common/thorhelper/roxiehelper.cpp

```diff
@@ -1648,6 +1646,31 @@ int readHttpHeaderLine(IBufferedSocket *linereader, char *headerline, unsigned m
     return bytesread;
 }
 
+void parseHttpParameterString(IProperties *p, const char *str)
+{
+    while (*str)
+    {
+        StringBuffer s, prop, val;
+        while (*str && *str != '&' && *str != '=')
+            s.append(*str++);
+        appendDecodedURL(prop, s.trim());
+        if (!*str || *str == '&')
+            val.set("1");
+        else
+        {
+            s.clear();
+            str++;
+            while (*str && *str != '&')
+                s.append(*str++);
+            appendDecodedURL(val, s.trim());
+        }
+        if (prop.length())
+            p->setProp(prop, val);
+        if (*str)
+            str++;
+    }
+}
```

Comment:
Correct - but I can't think of any scenario where you would need to support it.  But if this is to become a generic URL parser then it would be worth adding.